.github/workflows/cleanup.yml

```diff
@@ -68,3 +68,8 @@ jobs:
             echo "Deleting ${NAME}..."
             gcloud run revisions delete ${NAME} --project="$INTEGRATION_PROJECT_ID" --region="$INTEGRATION_REGION" --quiet --async
           done
+  key_cleanup:
+    uses: './.github/workflows/kms-key-cleanup.yml'
```

Comment:
What about just use the workflow instead of call a reusable work flow.

Reply:
Done